DA/Internal/Desugar.hs

```diff
@@ -150,6 +152,14 @@ toInterface = _toInterface
 
 class HasFromInterface t i where
   fromInterface : i -> Optional t
+  unsafeFromInterface : ContractId i -> i -> t
```

Comment:
Why do we need `unsafeFromInterface`?

Reply:
`DA/Internal/Desugar.hs` is just a stub file to allow us to test the fork on `Example.hs`. However, we haven't used it for anything other than templates until now, so I need to add definitions for for things that RdrHsSyn.hs generates for interfaces, such as `unsafeFromInterface`